testing/ncids-css-testing/stories/uswds-native/usa-collection/src/usa-collection.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Component from './usa-collection.twig';
+import FancyComponent from './usa-collection--fancy-date.twig';
+import MediaComponent from './usa-collection--media.twig';
+import HeadersComponent from './usa-collection--only-headers.twig';
+import { TestCase } from '../../../../components/test-case';
+import css from './index.scss';
+import img from '@nciocpl/ncids-css/uswds-img/sprite.svg';
+
+export default {
+	title: 'USWDS/Components/Collection',
+};
+
+
+const replaceContent = (content) => {
+	const template = HeadersTemplate.bind({})(content)
```

Comment:
don't need to pass content if there isn't any 

```suggestion
const replaceContent = () => {
	const template = HeadersTemplate.bind({})()
```